content/en/docs/tutorials/security/apparmor.md

```diff
@@ -250,10 +250,11 @@ error: error executing remote command: command terminated with non-zero exit cod
 
 To wrap up, let's look at what happens if we try to specify a profile that hasn't been loaded:
 
-```shell
-kubectl create -f /dev/stdin <<EOF
-```
+
+
+
 ```yaml
```

Comment:
```suggestion
```shell
```
Then it will be not a yaml.

Reply:
I think it's an total shell command and I'm confused about the meta data provided in command